pkg/cleaner/aws/aws.go

```diff
@@ -62,7 +87,33 @@ func (a *Cleaner) Clean() error {
 	return nil
 }
 
-func shouldBeDeleted(stack *cloudformation.Stack) bool {
+func (a *Cleaner) cleanBuckets() error {
+	input := &s3.ListBucketsInput{}
+	output, err := a.s3Client.ListBuckets(input)
+	if err != nil {
+		return err
+	}
+	for _, bucket := range output.Buckets {
+		a.logger.Log("level", "debug", "message", fmt.Sprintf("checking bucket %q", *bucket.Name))
+		if bucketShouldBeDeleted(bucket) {
+			a.logger.Log("level", "debug", "message", fmt.Sprintf("bucket %q should be deleted", *bucket.Name))
+			deleteBucketInput := &s3.DeleteBucketInput{
+				Bucket: bucket.Name,
+			}
+			_, err := a.s3Client.DeleteBucket(deleteBucketInput)
+			if err != nil {
+				// do not return on error, try to continue deleting.
+				a.logger.Log("level", "debug", "message", fmt.Sprintf("could not delete bucket %q: %#v", *bucket.Name, err))
+			} else {
+				a.logger.Log("level", "debug", "message", fmt.Sprintf("bucket %q was deleted", *bucket.Name))
```

Comment:
I feel like the logging is a bit inconcistent. Here as well we agreed on some conventions.

Reply:
Sure, I've tried to align with fmt, PTAL.